packages/pages-components/src/components/link/methods.ts

```diff
@@ -24,10 +30,14 @@ const getHref = (cta: CTA): string => {
  * Regex defined in HTML Spec for <input type="email"> validation.
  * https://html.spec.whatwg.org/#email-state-(type=email)
  */
-const isEmail = (string: string): boolean => {
+const isEmail = (link: string): boolean => {
+  if (link.includes("mailto:")) {
```

Comment:
probably a nit, but wouldn't this match `http://www.mysuperawesomewebsite?mailto:=foo`?

Reply:
Good call. Changed it to check using startsWith